# Generated by Django 2.2.24 on 2022-05-29 08:56

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('property', '0013_auto_20220527_1435'),
    ]

    operations = [
        migrations.RenameField(
            model_name='flat',
            old_name='like',
            new_name='likes',
        ),
        migrations.RenameField(
            model_name='owner',
            old_name='owners_phonenumber',
            new_name='phonenumber',
        ),
        migrations.RenameField(
            model_name='owner',
            old_name='owner_pure_phone',
            new_name='pure_phonenumber',
        ),
        migrations.RemoveField(
            model_name='complaint',
            name='flat_complaint',
        ),
        migrations.RemoveField(
            model_name='owner',
            name='owner_flat',
        ),
        migrations.AddField(
            model_name='complaint',
            name='flat',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='complaints', to='property.Flat', verbose_name='Квартира, на которую жаловались'),
        ),
        migrations.AddField(
            model_name='owner',
            name='flat',
            field=models.ManyToManyField(db_index=True, related_name='owners', to='property.Flat', verbose_name='Квартиры в собственности'),
        ),
        migrations.AlterField(
            model_name='complaint',
            name='user',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='complaints', to=settings.AUTH_USER_MODEL, verbose_name='Кто жаловался'),
        ),
    ]
